packages/cli/src/bin/dev-portal/PersonSideSheet/sheets/FeatureSheetContent.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { FeatureTogglerApp } from './FeatureTogglerApp';
+import { FeatureTogglerPortal } from './FeatureTogglerPortal';
+
+import { Divider, Icon, Button, Tabs } from '@equinor/eds-core-react';
+import { arrow_back, category } from '@equinor/eds-icons';
+Icon.add({ arrow_back, category });
+
+import type { SheetContentProps } from './types';
+
+export const FeatureSheetContent = ({ navigate }: SheetContentProps) => {
```

Comment:
short comment on what this component is ment todo

Reply:
Done